### PR TITLE
SUMO-227597: documentation for 12 webhook doc only apps

### PR DESCRIPTION
**Pull request by shivani-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -2356,6 +2356,18 @@
   "/cid/900": "/docs/beta",
   "/cid/9000": "/docs/integrations/app-development/jmx",
   "/cid/12323": "/docs/integrations/webhooks/buddy",
+  "/cid/12324": "/docs/integrations/webhooks/postman",
+  "/cid/12325": "/docs/integrations/webhooks/pusher",
+  "/cid/12326": "/docs/integrations/webhooks/rollbar",
+  "/cid/12327": "/docs/integrations/webhooks/sendGrid",
```
Comment (kimsauce) [inline cid-redirects.json:2362]:
```suggestion
  "/cid/12327": "/docs/integrations/webhooks/sendgrid",
```

File: docs/integrations/webhooks/sendgrid.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+id: sendGrid
```
Comment (kimsauce) [inline docs/integrations/webhooks/sendgrid.md:2]: No caps in URLs

```suggestion
id: sendgrid
```

File: sidebars.ts [hunk 3]
```diff
@@ -2241,6 +2241,18 @@ integrations: [
         collapsed: true,
         items: [
           'integrations/webhooks/buddy',
+          'integrations/webhooks/postman',
+          'integrations/webhooks/pusher',
+          'integrations/webhooks/rollbar',
+          'integrations/webhooks/sendGrid',
```
Comment (kimsauce) [inline sidebars.ts:2247]:
```suggestion
          'integrations/webhooks/sendgrid',
```

File: docs/integrations/webhooks/postman.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+id: postman
+title: Postman
+description: Learn about the collection process for the Postman integration.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/webhooks/postman-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Postman is an API development tool that streamlines API building, testing, and modification. It simplifies the process of working with APIs for developers. A webhook available in the Postman can forward events related to changes occurred in your Postman platform to the Sumo Logic HTTP endpoint. Using these logs, you can monitor API test results, collection changes, monitor alerts, environment modifications, and custom automation workflow triggers in Sumo logic. For more details, refer to the [Postman Documentation](https://learning.postman.com/docs/introduction/overview/).
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:10]:
```suggestion
Postman is an API development tool that streamlines API building, testing, modification, and simplifies the process of working with APIs for developers. You can use a webhook in the Postman platform to forward change events to the Sumo Logic HTTP endpoint. Using these logs, you can monitor API test results, collection changes, monitor alerts, environment modifications, and custom automation workflow triggers in Sumo logic. For more details, refer to the [Postman Documentation](https://learning.postman.com/docs/introduction/overview/).
```

File: docs/integrations/webhooks/postman.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Event types
+
+The Sumo Logic integration for Postman ingests Postman events into Sumo Logic through an outgoing webhook available in Postman. For more information on supported events that are ingested through the Postman webhook, see the [Postman documentation](https://learning.postman.com/docs/integrations/webhooks/).
+
+## Setup
+
+This section has instructions for collecting logs for the Sumo Logic's Postman webhook collection.
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:18]:
```suggestion
This section has instructions for collecting logs for the Sumo Logic Postman webhook collection.
```

File: docs/integrations/webhooks/postman.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+### Source configuration
+Follow the below steps to configure the Hosted Collector to receive Postman events:
+
+1. In the Sumo Logic portal, create a new [Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/) or use an existing one. Then add a [HTTP Logs and Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:23]:
```suggestion
1. In the Sumo Logic portal, create a new [Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/) or use an existing one. Then add an [HTTP Logs and Metrics Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).
```

File: docs/integrations/webhooks/postman.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+## Event types
+
+The Sumo Logic integration for Postman ingests Postman events into Sumo Logic through an outgoing webhook available in Postman. For more information on supported events that are ingested through the Postman webhook, see the [Postman documentation](https://learning.postman.com/docs/integrations/webhooks/).
+
+## Setup
+
+This section has instructions for collecting logs for the Sumo Logic's Postman webhook collection.
+
+### Source configuration
+Follow the below steps to configure the Hosted Collector to receive Postman events:
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:21]:
```suggestion
Follow the below steps to configure the Hosted Collector to receive Postman events.
```

File: docs/integrations/webhooks/rollbar.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/webhooks/rollbar-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Rollbar is an error tracking and monitoring platform that assists developers in identifying and resolving software issues quickly, enhancing application reliability. A webhook available in the Rollbar can forward events related to errors, occurrences, and activities from Rollbar platform to the Sumo Logic HTTP endpoint. Using these logs, you can monitor details such as error messages, stack traces, affected endpoints, user information, and environment variables in Sumo Logic. For more details, refer to the [Rollbar Documentation](https://docs.rollbar.com/docs).
+
+## Event types
+
+The Sumo Logic integration for Rollbar ingests Rollbar events into Sumo Logic through an outgoing webhook available in Rollbar. The following event types are ingested through the Rollbar webhook:
+- 10^nth occurrence
```
Comment (kimsauce) [inline docs/integrations/webhooks/rollbar.md:15]: @shivani-sumo should this be 10<sup>th</sup> ?

File: docs/integrations/webhooks/postman.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    - **Nickname**. Enter a Nickname for the integration.
+    - **Webhook URL**. Enter the Sumo Logic HTTP endpoint URL(source address) created above.
+    - For **Backup a collection** as the webhook type,
+        - **Choose workspace**. Select the workspace to which your collection belongs.
+        - **Choose Collection**. Select the collection you wish to send updates for.
+    - For **Post monitoring results** as the webhook type,
+        - **Choose workspace**. Select the workspace to which your collection belongs.
+        - **Choose Monitor**. Select the monitor you wish to send results for.
+6. Click **Add Integration**.
+7. Verify Postman events are getting ingested in Sumo Logic by executing the following query on Sumo Logic's search panel.
```
Comment (kimsauce) [inline docs/integrations/webhooks/postman.md:45]: Should this be ... Sumo Logic's **Log Search** panel?

**Review by kimsauce — CHANGES_REQUESTED** [review]
A few questions

File: docs/integrations/webhooks/rollbar.md (hunk 8 above)
Comment (shivani-sumo) [reply docs/integrations/webhooks/rollbar.md:16]: I have kept it as a `10^nth occurrence` by taking reference from the official documentation: https://docs.rollbar.com/docs/webhooks#configuration

Comment (kimsauce) [reply docs/integrations/webhooks/rollbar.md:16]: Thanks for confirming!

File: docs/integrations/webhooks/postman.md (hunk 9 above)
Comment (shivani-sumo) [reply docs/integrations/webhooks/postman.md:49]: Yes, It should be as a **Log Seach** panel. I am updating all 12 docs.

File: docs/integrations/webhooks/postman.md (hunk 4 above)
Comment (shivani-sumo) [reply docs/integrations/webhooks/postman.md]: I'll update all the 12 doc referring this.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**